### ISSUE TITLE
Use shared @edx/browserslist-config in frontend repos

### ISSUE THREAD

**Issue by adamstankiewicz** [openedx/frontend-wg#47]
Since we've decided to officially drop IE 11 support, one of the first things we can tackle is removing IE 11 (and other older browsers) from the `browserslist` config specified in package.json files throughout the platform for frontend facing repos.

See [this document](https://docs.google.com/spreadsheets/d/1fduFbAxIbckiwHKNCtc41ImAL_CFIl9hZcC4AjDkuvI/edit#gid=0) for list of PRs incorporating the shared `@edx/browserslist-config` NPM package that will need to be reviewed/merged.

**Note that these PRs are dependent on upgrading their respective repos to Node 16, else later JS syntax not available in Node 12 will cause Jest tests to fail.**

**Comment by arbrandes** [comment]
@Mashal-m, @abdullahwaheed, do you mind if we start linking PRs such as https://github.com/openedx/frontend-app-profile/pull/649 back here?  I had trouble finding which issue it was for after the fact.

**Comment by arbrandes** [comment]
Also, is there a separate issue for the Paragon version upgrade?

**Comment by Mashal-m** [comment]
> @Mashal-m, @abdullahwaheed, do you mind if we start linking PRs such as [openedx/frontend-app-profile#649](https://github.com/openedx/frontend-app-profile/pull/649) back here? I had trouble finding which issue it was for after the fact.

I have linked this ticket with respective PRs, sorry for the inconvenience.

**Comment by arbrandes** [comment]
That's great, thanks @Mashal-m!

**Comment by arbrandes** [comment]
Are we only missing the special-exams one?

**Comment by Mashal-m** [comment]
> Are we only missing the special-exams one?

yes, after merging the lib special exam, we will be done with this ticket.
